README.md

```diff
@@ -54,6 +54,15 @@ npm install
 npm test
 ```
 
+## Run example
+
+```
+python3 -m http.server
+
+```
+
+[StackOverflow: Start a server](https://stackoverflow.com/a/21608670)
+
```

Comment:
I think starting a local server is up to the developer. I use the `http-server` npm package myself.

```suggestion
```

Reply:
You can ignore this part. I put it to help me.